tests/HtmlTest.php

```diff
@@ -1,416 +1 @@
-<?php
```

Comment:
I didn't touch this file, it is still ok in mine locally, so I don't understand why this appears like that